CONTRIBUTING.md

```diff
@@ -1,6 +1,8 @@
 # Contributing to PyScript
 
-Thank you for wanting to contribute to the PyScript project!
+Thank you for your interest in contributing to the PyScript project! This document provides guidelines on how to contribute to the project effectively and what to expect from the project's development flow and guidelines.
+
+Please read these guidelines thoroughly before opening a pull request, issue, or discussion.
```

Comment:
The fact that they won't doesn't mean they shouldn't 😅 

This document service is really for everyone (maintainers, contributors, etc..) to the way we work and communicate (in terms of principles, rules, guidelines, etc...). If they want to contribute, the least they can do is check it out. In general terms, I do agree with you that most people won't read but, the idea here is that at least we do have written rules that we can look at, if people are not doing it it's their fault (not because the project has clear written guidelines).